Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM python:3.6
+RUN pip install asreview
```

Comment:
It might be interesting to add our plugin `asreview-covid19` there as well.

Reply:
Yes, I thought about it.

Same for other plugins.

but then I saw that they all have different versions,
so it would be hard to make a versioned Docker tag.

But maybe it makes sense to have only a single Docker image,
which includes all plugins in the latest released version.

I will have a look at that.

Reply:
I think the best is to offer 2 Dockerfile and images.

One for the GUI app, where the web app is the entrypoint, like the current one.
Adding there covid19 plugin is fine.

A second Dockerfile  + image which is ment to be used from commandlie.
This could then have all plugins, but the entrypoint should be either /bin/bash or "asreview"

Mabe names could be

asreview-app
asreview-cli

I make an other issue for this